docker-compose.yml

```diff
@@ -13,6 +13,8 @@ crossdock:
 
       - BEHAVIOR_DANCE=client,server,transport
       - BEHAVIOR_SING=client,server,transport
+      - BEHAVIOR_RUN=client,server,transport
+      - BEHAVIOR_SKIP_RUN=client=alpha+server=alpha
```

Comment:
Actually, we wanted to maintain same word "skip" and provide additional reason to why we were skipping. I'd like to stick to SKIP_ but I'm flexible if others feel the other way.